doc/rippled-example.cfg

```diff
@@ -987,23 +972,6 @@ pool.ntp.org
 [ips]
 r.ripple.com 51235
 
-# Public keys of the validators that this rippled instance trusts.  The latest
```

Comment:
You've removed the `[validation_quorum]` and `[validators]` entries from the default config file, which makes it more difficult for someone to quickly bootstrap a server; marginally so, but still more difficult: two files need to be copied and renamed to have a functional server. This is a bit of a concern for me.